spl/src/associated_token.rs

```diff
@@ -3,9 +3,8 @@ use anchor_lang::solana_program::pubkey::Pubkey;
 use anchor_lang::Result;
 use anchor_lang::{context::CpiContext, Accounts};
 
-pub use spl_associated_token_account::{
-    get_associated_token_address, get_associated_token_address_with_program_id, ID,
-};
+pub use spl_associated_token_account;
+pub use spl_associated_token_account::ID;
```

Comment:
Can we also keep `get_associated_token_address` and `get_associated_token_address_with_program_id` for now so that this is not a breaking change? Planning to make a patch release soon. CI also has [errors](https://github.com/coral-xyz/anchor/actions/runs/9333801827/job/25691263973?pr=2999#step:10:141) due to this.